Dockerfile

```diff
@@ -19,9 +19,16 @@ COPY ./src ./src
 
 RUN cargo build --release
 
-# Stage 2: Create final Docker image with debain-slim.
+# Stage 2: Create final Docker image with debian-slim.
 FROM ghcr.io/linuxcontainers/debian-slim:12.5
 
+# Augment debian-slim with tools needed to run in nextflow
+RUN apt-get update && \
+    apt-get --no-install-recommends install -y --force-yes \
+      procps && \
```

Comment:
please fix linter errors

Reply:
@sjlangley I think the issues are sorted, can you trigger the CI checks? (I classified it as `feat:` rather than `fix:`, but that kinda depends on whether you expect that it should have originally worked).